app/services/audit/catalog_to_archive.rb

```diff
@@ -15,6 +15,15 @@ def self.check_child_zip_part_attributes(zmv, results)
         return false # everything else relies on checking parts, nothing left to do
       end
 
+      total_part_size = zmv.total_part_size
+      moab_version_size = zmv.preserved_object.complete_moab&.size || 0
```

Comment:
@jmartin-sul I'm not entirely confident I've got the right sizes here, but maybe I'm not far off?

Reply:
@mjgiarlo I think we probably want a method like `CompleteMoab#version_size_from_disk(version)`, that'll use the `complete_moab.moab_storage_root.storage_location` and the druid (from the parent `PreservedObject`) to get a druid tree (for which i think you can use `DruidTools::Druid.new(druid, storage_location).path`).  And then this new method would get the size of e.g. the `v0003` dir in the particular Moab.  Sorry, when we were pairing on this yesterday I mentioned thinking that the sizes in the DB could be trusted, but a bit of nuance is that the `CompleteMoab` encompasses all versions, so its size field isn't at the right level of granularity for comparison with one `ZippedMoabVersion`, I don't think. (perhaps replacing `CompleteMoab#size` with a `CompleteMoab#version_sizes` array is worth ticketing and considering, since that might be useful info for querying here and in general)

Reply:
@jmartin-sul Sounds good, and rings a bell, even. 😄 I'll tackle that next.